.github/workflows/ci.yml

```diff
@@ -111,7 +116,12 @@ jobs:
         run: yarn build
 
       - name: Run Playwright tests
-        run: npx playwright test --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
+        run: npx playwright test --grep-invert CHROME_ONLY --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
+
+      # Some tests are only run on Chrome, marked with CHROME_ONLY in their name
+      - name: Run Chrome-only Playwright tests
+        run: npx playwright test --grep CHROME_ONLY --project='chromium'
+        if: matrix.shardIndex == 1
```

Comment:
Idea: Let's run these tests (chrome_only and reverse) in parallel to speed up CI

Reply:
We can, but the chrome-only tests currently include just one and it takes around 20-25s, I don't see us adding more anytime soon. AFAIK, parallelizing them would mean creating a separate job, mostly identical to the current Playwright test one: 

- Less readable YAML setup that now includes a reusable job/workflow with some parameters
- The new job is an extra 2-3 minutes of CI time (parallel but still) to set everything up (yarn install, yarn build, playwright install) just to save 20s. 

CI runtimes can vary a minute up or down just randomly, is it worth the extra resources/complexity to save 20s? Am I missing a simpler way of doing it?

Reply:
Let's leave it for now, it doesn't make sense to make it more complicated for these additional seconds :)